lm20/pipelines.py

```diff
@@ -168,27 +168,50 @@ def file_path(self, request, response=None, info=None, *, item=None):
 
         media_guid = hashlib.sha1(to_bytes(request.url)).hexdigest()
 
-        content_disposition = headers.get("Content-Disposition").decode()
+        content_disposition = headers.get("Content-Disposition")
+        try:
+            # Occassionally comes in as a bytes object
+            content_disposition = content_disposition.decode()
+        except AttributeError:
+            pass
+
+        if content_disposition:
+            content_type = headers.get("Content-Type")
+            try:
+                # Occassionally comes in as a bytes object
+                content_type = content_type.decode()
+            except AttributeError:
+                pass
```

Comment:
i would pass content_disposition to get_media_ext and do all this checking and stuff in there.

make it solely the get_media_ext's responsibility to return the media_ext instead of spreading that responsibility around.